### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by sckott** [comment]
thanks!

File: _posts/2019-03-04-control-flow-exceptions.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ But also we return the response object (`HttpResponse` from the `crul` package i
...
 
-This approach is fine if your users are more advanced, but most packages/libraries are probably trying to give back a data object that users are familiar with. In R, that is clearly the data.frame. When there is a 400/500 series error, one option is to return an empty data.frame and throw a warning about the error, hopefully with enough information for the user to fix the request. This is probably best for naive users, but any package has some more advanced users that would benefit from more information; and more information will help a naive user + the maintainer debug a problem eaiser.
+This approach is fine if your users are more advanced, but most packages/libraries are probably trying to give back a data object that users are familiar with. In R, that is clearly the data.frame. When there is a 400/500 series error, one option is to return an empty data.frame and throw a warning about the error, hopefully with enough information for the user to fix the request. This is probably best for naive users, but any package has some more advanced users that would benefit from more information; and more information will help a naive user + the maintainer debug a problem easier.
```
Comment (sckott) [inline _posts/2019-03-04-control-flow-exceptions.md]: I don't see any change here, what was changed?

Comment (maelle) [reply _posts/2019-03-04-control-flow-exceptions.md]: yeah the diff is bad. eaiser became easier (last word)

File: _posts/2019-03-04-control-flow-exceptions.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ Even if exceptions are a slowish procedure, there is an argument to be made that
...
+I'm not sure if I'll change anything in packages I maintain or not. I'll keep thinking about this and ask around to gauge others opinions on this. Part of me wants to follow the avoid exceptions path, but I worry about two things. First, the complexity increases for me as the developer. If I don't fail out, then I have to deal with parsing somehow every response. It's not as simple as giving back the HTTP response; I ideally want to give users a data structure they are familiar with, i.e., a data.frame. Second, for the user, if I give back a list or an `R6` object, that increases complexity on their side. Is the benefit of more information worth the cost of more complexity for the user? I've no idea.
```
[the remark this answers is not in the document]
Comment (maelle) [reply _posts/2019-03-04-control-flow-exceptions.md]: about in " I'll keep thinking aobut"

[the remark this answers is not in the document]
Comment (sckott) [reply _posts/2019-03-04-control-flow-exceptions.md]: thanks!

**Pull request merged**